### DISCUSSION TITLE
Introduction to WebApplicationFactory

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#24] [General]
# Introduction to WebApplicationFactory

This article will show you what exactly a WebApplicationFactory is and why it is so "mighty" when it comes down to testing. I will highlight some of the properties, which you can leverage to write powerful tests.

https://steven-giesel.com/blogPost/cd62475b-2c7d-4ce2-bd97-9670f91ebac8

**Comment by majumajid** [comment]
Love it :) great

**Reply by linkdotnet** [reply]
Thank you @majumajid for the support!

**Comment by alexssantos** [comment]
Really nice job. man!

**Reply by linkdotnet** [reply]
Thanks for the kind words! Much appreciated
